parser_uk.go

```diff
@@ -0,0 +1,34 @@
+package whoisparser
+
+import (
+	"regexp"
+)
+
+var ukParser = &Parser{
+
```

Comment:
Please be consistent about your code style and empty lines.

Reply:
Done